src/main/scala-2.11/markets/exchanges/ExchangeLike.scala

```diff
@@ -1,13 +1,16 @@
 package markets.exchanges
 
 import akka.actor.ActorRef
+import akka.agent.Agent
 
-import markets.{BaseActor, MarketActor}
+import markets.orders.Order
+import markets.tickers.Tick
+import markets.{BaseActor, Cancel, MarketActor}
 import markets.clearing.engines.MatchingEngine
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1560525571/issues/source?bid=2921361&fileBranchId=3019109#l9)